Sezai/index.js

```diff
@@ -0,0 +1,121 @@
+//These exercises are aimed at making you more comfortable with using conditionals as well as operators. Use "if" statements to complete the following exercises. Print your results to the console.
+
+//Check if two given integer values are in the range 50 to 99 (this includes both 50 and 99). Print true if either of them are in the range.
+
+
+let num1 = 49;
+let num2 = 100;
+
+if(num1 >=50 && num2 <= 90){
+    console.log("they are in the range of 50 and 90 ==> ");
+}else {
+    console.log("they are NOT in the range of 50 and 90");
+}
+
+
+//Check if three given integer values are in the range 50 to 99 (inclusive). Print true if one or more of them are in the range.
+
+let num3 = 100;
+
+if(num1>=50 || num2 <=99 || num3 <=99){
+    console.log("they are in the range of 50 and 99");
+} else{
+    console.log("they are NOT in the range of 50 and 99");
+}
+
+//  Declare the variables a, b and c, and give them number values. Check which one out of the three variables has the largest value and print it to the console. Then change the values of the variables to see if your conditional still works.
+
+let a=33
+let b=66666
+let c=99
+
+let highestValue = Math.max(a,b,c);
+console.log("highest Value", highestValue);
+
+
+//Create a new string adding "Py" in front of a given string. If the given string begins with "Py" then print the original string.
+
+let givenString = "Py cats are cute, but dogs also"
```

Comment:
Here you need 2 variables, the value of first one(given string) is up to you but the second one should be "Py" and then check if the given string begins with Py using if statement